src/users/register.js

```diff
@@ -14,17 +14,47 @@ export function register(user /** async */) {
     url         : Urls.userRegister(),
     isAsync     : isAsync,
     asyncHandler: responder && wrapAsync(responder),
-    data        : user
+    data        : enrichWithLocaleInfo(user)
   })
 
   return isAsync ? result : parseResponse(result)
 }
 
+function enrichWithLocaleInfo(user) {
+  if (user.blUserLocale === null || user.blUserLocale === '') {
+    return user
+  }
+
+  user.blUserLocale = getClientLanguage()
```

Comment:
what if user want to setup the `blUserLocale` property manually? look like it's not possible, isn't it?


what if we will detect/setup `blUserLocale` property only when there is no value
 
```
function enrichWithLocaleInfo(user) {
  if (!user.blUserLocale) {
   user.blUserLocale = getClientLanguage()
  }

return user
```

Reply:
Changed next way:
https://github.com/Backendless/JS-SDK/pull/138/commits/8466f400b846aa39113143b59a161911ba7aeb6f

`user.blUserLocale` may be undefined, according to ticket description.

Reply:
Fixed